[FW][FIX] portal_timesheet: fix access of internal users

fw-bot-adhoc opened the pull request:
Forward-Port-Of: ingadhoc/miscellaneous#105

roboadhoc commented:
[Pull request status dashboard](https://runbot.adhoc.com.ar/ingadhoc/miscellaneous/pull/106).

fw-bot-adhoc commented:
@matiasperalta1 @nicomacr cherrypicking of pull request ingadhoc/miscellaneous#105 failed.

* 8b5c880195576a5013860fa6e37424d75dc9426b <- on this commit
* 123f2164cad209da9f9ee6c1f59e01bc785249ff

stdout:
```
CONFLICT (modify/delete): portal_timesheet/security/portal_timesheet.xml deleted in HEAD and modified in 8b5c880... [FIX] portal_timesheet: fix access of internal users. Version 8b5c880... [FIX] portal_timesheet: fix access of internal users of portal_timesheet/security/portal_timesheet.xml left in tree.
CONFLICT (modify/delete): portal_timesheet/__manifest__.py deleted in HEAD and modified in 8b5c880... [FIX] portal_timesheet: fix access of internal users. Version 8b5c880... [FIX] portal_timesheet: fix access of internal users of portal_timesheet/__manifest__.py left in tree.

```

stderr:
```
18:17:45.954621 git.c:439               trace: built-in: git cherry-pick 8b5c880195576a5013860fa6e37424d75dc9426b
error: could not apply 8b5c880... [FIX] portal_timesheet: fix access of internal users
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
----------
status:

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

nicomacr commented:
@matiasperalta1 podes revisar los conflictos?
